docs/Documentation/Applications/vasp.md

```diff
@@ -452,7 +446,27 @@ To run VASP, the following 4 input files are needed: POSCAR, POTCAR, INCAR, KPOI
     # If the multi-node calculations are breaking, replace the srun line with this line
     # I_MPI_OFI_PROVIDER=tcp mpirun -iface ens7 -np 60 vasp_std
     ```
-??? example "Vermilion: VASP 6 (OpenACC) on GPUs"
+
+??? note "Performance Notes"
+
+    On Vermilion, VASP runs more performantly on a single node. Many issues have been reported for running VASP on multiple nodes, especially when requesting all available cores on each node. In order for MPI to work reliably on Vermilion, it is necessary to specify the interconnect network that Vermilion should use to communicate between nodes. If many cores are needed for your VASP calculation, it is recommended to run VASP on a singe node in the lg partition (60 cores/node), which provides the largest numbers of cores per node and use the following settings that have been shown to work well for multi-node jobs on 2 nodes. The Open MPI multi-node jobs are more reliable on Vermilion, but Intel MPI VASP jobs show better runtime performance as usual.
+    
+    If your multi-node **Intel MPI VASP** job is crashing on Vermilion, try replacing your srun line with the following mpirun run line. ```-iface ens7``` sets ens7 as the interconnect. 
+    ```
+    I_MPI_OFI_PROVIDER=tcp mpirun -iface ens7 -np 16 vasp_std
+    ```
+
+    If your multi-node **Open MPI VASP** job is crashing on Vermilion, replace a call to load an openmpi module with the following lines. The OMPI_MCA_param variable sets ens7 as the interconnect. 
+
+    ```
+    module use /nopt/nrel/apps/220525b/level01/modules/lmod/linux-rocky8-x86_64/gcc/12.1.0
+    module load openmpi
+    OMPI_MCA_param="btl_tcp_if_include ens7"
+    ```
+
+#### GPU
+
+??? example "Sample job script: Vermillion - VASP 6 (OpenACC)"
   
     ```
     #!/bin/bash
```

Comment:
Has the module for this example changed?  I get an error loading the module (with or without sourcing the standard /nopt/nrel/apps env).

```
[jmcfarla@vs-login-1 vasp-docs]$ ml vasp/6.3.1-nvhpc_acc
Lmod has detected the following error: The following module(s) are unknown: "vasp/6.3.1-nvhpc_acc"
[jmcfarla@vs-login-1 vasp-docs]$ source /nopt/nrel/apps/210929a/myenv.2110041605                                  
[jmcfarla@vs-login-1 vasp-docs]$ ml vasp/6.3.1-nvhpc_acc
Lmod has detected the following error: The following module(s) are unknown: "vasp/6.3.1-nvhpc_acc"
```

Reply:
I've created an issue to review all the vermillion modules.